Safe/src/main/java/org/openintents/safe/DBHelper.java

```diff
@@ -54,17 +54,30 @@
     private static final String TABLE_SALT = "salt";
     private static final String TABLE_PACKAGE_ACCESS = "package_access";
     private static final String TABLE_CIPHER_ACCESS = "cipher_access";
+    private static final String CREATE_TABLE = "create table ";
+    private static final String ID_INTEGER_PRIMARY_KEY_AUTOINCREMENT = "id integer primary key autoincrement, ";
+    private static final String DROP_TABLE = "drop table ";
+    private static final String VERSION = "version";
+    private static final String SQLITE_EXCEPTION = "SQLite exception: ";
+    private static final String DESCRIPTION = "description";
+    private static final String WEBSITE = "website";
+    private static final String USERNAME = "username";
+    private static final String PASSWORD = "password";
+    private static final String CATEGORY = "category";
+    private static final String UNIQUE_NAME = "unique_name";
+    private static final String LAST_DATE_TIME_EDIT = "lastdatetimeedit";
+    
     private static final int DATABASE_VERSION = 4;
     private static String TAG = "DBHelper";
     Context myCtx;
 
     private static final String DBVERSION_CREATE =
-            "create table " + TABLE_DBVERSION + " ("
+            CREATE_TABLE + TABLE_DBVERSION + " ("
```

Comment:
Could you revert these changes for sql string building.They should be as plain text as possible or using a Builder. I am for plain text.